<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CASE_SWDEP</string>
  </property>
  <property name="accessibleName">
   <string comment="Студент"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="tableRadio">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>Добавить таблицу</string>
    </property>
    <property name="statusTip">
     <string>Добавить таблицу</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/table_add.ico</normaloff>:/ico/table_add.ico</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="one_to_one">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Связь 1:1</string>
    </property>
    <property name="statusTip">
     <string>Связь 1:1</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>1:1</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/Link-512.png</normaloff>:/ico/Link-512.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="Editing">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Перемещение таблиц</string>
    </property>
    <property name="statusTip">
     <string>Перемещение таблиц</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/1024px-ArrowCross.svg.png</normaloff>:/ico/1024px-ArrowCross.svg.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="one_to_N">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Связь 1:N</string>
    </property>
    <property name="statusTip">
     <string>Связь 1:N</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>1:N</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/Link-512.png</normaloff>:/ico/Link-512.png</iconset>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>961</width>
      <height>591</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>942</width>
       <height>572</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioMause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Мышь</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/arrow-cursor-mouse-pixel.png</normaloff>:/ico/arrow-cursor-mouse-pixel.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Работа с таблицами</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Связи</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDelete">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Удаление таблицы</string>
    </property>
    <property name="statusTip">
     <string>Удаление таблицы</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/table-delete.png</normaloff>:/ico/table-delete.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMove">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/ico/1024px-ArrowCross.svg.png</normaloff>:/ico/1024px-ArrowCross.svg.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Фаил</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="createSript"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить как..</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="createSript">
   <property name="text">
    <string>Создание скрипта</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Создание  базы данных</string>
   </property>
  </action>
  <action name="close">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
